assets/js/atomic/blocks/product-elements/button/block.js

```diff
@@ -116,14 +151,22 @@ const AddToCartButton = ( { product } ) => {
 		<ButtonTag
 			aria-label={ buttonAriaLabel }
 			className={ classnames(
+				'wc-block-components-product-button__button--global-style',
```

Comment:
Why do we need another class here? Scanning this PR, I see it's for Global Style. Is there any issue with the existing class (`wc-block-components-product-button__button`)?

Reply:
I was convinced that this resolved the problem about the font size :(

I did another cycle of tests and I noticed that this is not true :/ 
I removed this new class and thanks for bringing up this!